bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -44,6 +46,10 @@
   </div>
   {% if show_desktop %}
     {# unrecognized/unsupported platform #}
+    <p class="windows-xpvista-download">
+      <span class="icon" aria-hidden="true"></span>
+      {{ _('<a href="%(url)s">Mozilla no longer provides security updates for Firefox on Windows XP or Vista</a>, but you can still download the final Windows 32-bit version below.')|format(url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}
```

Comment:
From an SEO point of view we're adding this text numerous times to just about every page on the website. I'm not sure if there's any real impact here, but I'd be curious to know if @rraue has any thoughts here?

The vast majority of pages on mozorg no longer serve first-class CSS & JS to IE < 9, with the only real exception being our main download page on `/firefox/new/`. Another option could be to add it only to `/firefox/new/`, as opposed to everywhere.

Reply:
Less text, not being related to the actual page, in the body is better than more, I agree @alexgibson. On the firefox pages it doesn't harm anything as it is text related to the topic of the page.

Reply:
I'd be interested in @cpeterso's thoughts here. Is this message really needed everywhere, or just on our main download pages? e.g. `/firefox/new/`, `/firefox/`, `/firefox/all/`?

Reply:
I'm not sure if it can easily be repurposed, not at least while the existing banner is running.